.github/workflows/check-odo.yml

```diff
@@ -0,0 +1,43 @@
+name: check-odo
+
+on: [workflow_dispatch]
+
+jobs:
+  check-odo-repo:
+    runs-on: ubuntu-latest
+    env:
+        TOOL_REPO: redhat-developer/odo
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Check Out Code
+        uses: actions/checkout@v2
+      - name: Get latest ODO version
+        run: |
+          echo "REPO_ODO_VERSION=$(cat src/tools.json | jq -r .odo.version)" >> $GITHUB_ENV
+          LATEST_TOOL_RELEASE_RESP=$(gh release --repo ${{ env.TOOL_REPO }} view --json tagName,url)
+          echo "LATEST_TOOL_RELEASE=$(echo ${LATEST_TOOL_RELEASE_RESP} | jq -r .tagName | sed 's|v||')" >> $GITHUB_ENV
+          echo "LATEST_TOOL_URL=$(echo ${LATEST_TOOL_RELEASE_RESP} | jq -r .url)" >> $GITHUB_ENV
+      - name: Find existing PR for ODO version
+        run: |
+          echo PR_URL=$(gh pr --repo ${{ github.repository }} list --state all --search "update odo ${{env.LATEST_TOOL_RELEASE}} in:title" --json url) >> $GITHUB_ENV
```

Comment:
@msivasubramaniaan notes that this would return `[]` even with no matches so the `PR_URL` checks would never be empty. I think adding `| jq -r '.[].url` should do what we need here, at the end of the line.

Reply:
For handling array, I am using to get it [length](https://github.com/msivasubramaniaan/vscode-openshift-tools/blob/61e38ce97db57987237174dc554e38d18b8fafc3/.github/workflows/check-crc.yml#L25) and made a condition based on it. It will prevent to create another PR

Reply:
I went with `| jq length`. I think because the result is always an array, it should be safe to use. Let me know if there's a reason I missed for doing `. | length`.

Reply:
As far as it returns an array I don't feel `. | length` is not needed and `jq length` enough